examples/perfmanager/main.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This example program shows how the `view` and `property` packages can
+be used to navigate a vSphere inventory structure using govmomi.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/simulator"
+	"github.com/vmware/govmomi/view"
+	"github.com/vmware/govmomi/vim25/methods"
+	"github.com/vmware/govmomi/vim25/mo"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// ESXi model + initial set of objects (VMs, network, datastore)
+	model := simulator.ESX()
+
+	defer model.Remove()
+	err := model.Create()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	s := model.Service.NewServer()
+	defer s.Close()
+
+	// Connect and login to ESX or vCenter
+	// Using simulator instead of real ESX/vCenter
+	//c, err := examples.NewClient(ctx)
+	c, err := govmomi.NewClient(ctx, s.URL, true)
```

Comment:
I'd also prefer to stay consistent with the other examples, and use `examples.NewClient` instead.  I do like the idea of defaulting to vcsim if the `url` flag isn't set.